src/javax/microedition/io/file/FileSystemRegistry.java

```diff
@@ -18,11 +18,15 @@
 
 import java.util.Enumeration;
 
-public abstract class FileSystemRegistry 
+public class FileSystemRegistry extends Object
```

Comment:
`extends Object` is unnecessary as all Java classes extend Object.

Reply:
Yep, that was just to make an exact transcription of the docs, i still need to clean some things up.